src/TouristInfoCards.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
This component is responsible for rendering all the Tourist Info cards. Could we have a component that is responsible for rendering a single card?

Reply:
Yes it's a good idea. I made a new component called Card and passed the city as a prop for it